prow/scripts/lib/gardener/aws.sh

```diff
@@ -84,6 +84,16 @@ gardener::provision_cluster() {
     )
 }
 
+gardener::alpha_deploy_kyma() {
+  log::info "Deploying Kyma"
```

Comment:
For now Kyma deployment works the same for every cluster created through Gardener (AWS, GCP, Azure). Wouldn't it be more readable and maintainable to implement this function in another file and import it when needed for every provider-specific script?

Reply:
If we keep only one version of this function then I think we should put in https://github.com/kyma-project/test-infra/blob/master/prow/scripts/lib/kyma.sh

Reply:
atm idk if we will have only one version or many, i guess it was not tested yet. i have no problem moving it to a separate file then, if we will see a need to have specifics for providers then we can implement it.